CRM/Profile/Page/View.php

```diff
@@ -140,12 +133,10 @@ public function preProcess() {
     }
```

Comment:
@mlutfy what about the listingURL being assigned just above this line (click the + to expand the diff in github)

Reply:
@colemanw Doesn't make much sense to me, but most of the Profiles code reads to me as "huh, does this really work", and it sometimes surprises me when it does.

I don't know when would the `gid` (profile group) be null? and I've never encountered  `/civicrm/profile?force=1`  ex: 

- https://civicrm.org/civicrm/profile?force=1 fatals on gid=77 for some reason
- https://ml1.dev501.symbiodev.xyz/civicrm/profile?force=1 fatals on an SQL error

and that's just the page being linked to. Still, feels a bit like a rabbit hole.

Reply:
All the same, best remove the `assign` from there for consistency?

Reply:
Right ok - done